static/src/javascripts/projects/commercial/modules/dfp/render-advert.ts

```diff
@@ -18,9 +18,20 @@ import { renderAdvertLabel } from './render-advert-label';
  *
  */
 
+/**
+ * Types of events that are returned when executing a size change callback
+ */
+type CallbackEvent = {
```

Comment:
Having looked at this again I _think_:

- the `callback` is only ever a `slotRenderEndedEvent` invoked here:
https://github.com/guardian/frontend/blob/e6bc413fbc2017d133dedaf75c814d52f77846c6/static/src/javascripts/projects/commercial/modules/dfp/render-advert.ts#L265

- also I think we can improve the code quite a bit by swapping the callback parameter order so the `event` is an optional second parameter. In which case it does not need to be declared by all the functions that don't use it. The only function that does use it seems to be `outOfPageCallback`